okio/src/main/java/okio/HashingSource.java

```diff
@@ -57,6 +57,11 @@ public static HashingSource sha256(Source source) {
     return new HashingSource(source, "SHA-256");
   }
 
+  /** Returns a sink that uses the SHA-512 hash algorithm to produce 512-bit hashes. */
+  public static HashingSource sha512(Source source) {
```

Comment:
Thanks!

Reply:
Merged this commit to master ahead of the extensions.